src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Rectangle extends Default {
+    private double side;
+    private double firstLeg;
+    private double secondLeg;
+
+    public Rectangle(String color, double side, double firstLeg, double secondLeg) {
+        super(color);
+        this.side = side;
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getProperty() {
+        return firstLeg * secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
```

Comment:
The method 'getArea()' calculates the area using 'side * side', which is incorrect for a rectangle. It should use 'length * width' or equivalent variables.